WordPress/Tracks+ShareExtension.swift

```diff
@@ -43,6 +43,15 @@ extension Tracks {
         trackExtensionEvent(.categoriesSelected, properties: properties as [String: AnyObject]?)
     }
 
+    public func trackExtensionPostTypeOpened() {
+        trackExtensionEvent(.postTypeOpened)
+    }
+
+    public func trackExtensionPostTypeSelected(_ postType: String) {
+        let properties = ["post_type_tags": postType]
```

Comment:
Ah, I meant to bring this up to you. Are these defined anywhere other than in the app (the strings to identify tracking events)? I have just added ones following an existing pattern but I'm unsure if thats enough.

The "_tags" part matches the other track events:

```
 let properties = ["categories_tags": categories]
```
I presume it's to identify the dict as a set of tags pertaining to each element? Or perhaps it's an error on that "categories" one, too, since tags are just:

```
let properties = ["selected_tags": tags]
```

Reply:
> Are these defined anywhere other than in the app (the strings to identify tracking events)?

Not really.  They're captured by our tracking tools, but they're defined by the App.

In the case of categories, I guess it's possible to think of categories as category-tags.  But this is just a theory as I've no recollection of the naming of that property.

IMHO we can safely name this property just:

"post_type"